secondary_bulk_insert_script.sql

```diff
@@ -0,0 +1,3 @@
+/*
+* This file is for our insertions into the BD tables after sample data has executed
```

Comment:
I think it should be DB instead of BD? Otherwise perfect :)

Reply:
Yep.